mergin/client.py

```diff
@@ -990,3 +992,15 @@ def has_writing_permissions(self, project_path):
         """
         info = self.project_info(project_path)
         return info["permissions"]["upload"]
+
+    def clean_temp_files(self, directory: str) -> None:
+        """
+        Removes all files matchning patter "*-diff-*" from .mergin folder inside specified folder.
+
+        :param directory: project's local path
+        :type directory: str
+        """
+        mergin_dir = Path(directory) / ".mergin"
+        if mergin_dir.exists():
+            for file in mergin_dir.glob("*-diff-*"):
+                file.unlink()
```

Comment:
Rather than doing this kind of bulk erase, I would prefer that after the call to get_push_changes(), we figure out list of all temporary diff files (using `changes["updated"]` entries that have "diff" key - as created in get_push_changes().